template.md

```diff
@@ -0,0 +1,305 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: [MANAGER ALIAS]
+ms.date: [CREATION/UPDATE DATE]
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.assetid: [GET ONE FROM guidgenerator.com]
+
+# optional metadata
+
+#ROBOTS:
+#audience:
+#ms.devlang: dotnet
+#ms.reviewer: [ALIAS]
+#ms.suite: ems
+#ms.tgt_pltfrm:
+#ms.custom:
+
+---
+
+# Metadata and Markdown Template
+
+This core-docs template contains examples of Markdown syntax, as well as guidance on setting the metadata. To get the most of it, you must view both the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md) and the [rendered view](https://github.com/dotnet/core-docs/blob/master/template.md) (for instance, the raw Markdown shows the metadata block, while the rendered view does not).
+
+When creating a Markdown file, you should copy this template to a new file, fill out the metadata as specified below, set the H1 heading above to the title of the article, and delete the content. 
+
+
+## Metadata 
+
+The full metadata block is above (in the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md)), divided into required fields and optional fields. Some key notes:
+
+- You **must** have a space between the colon (:) and the value for a metadata element.
+- If an optional metadata element does not have a value, comment out the element with a # (do not leave it blank or use "na"); if you are adding a value to an element that was commented out, be sure to remove the #.
+- Colons in a value (for example, a title) break the metadata parser. In their place, use the HTML encoding for a colon of `&#58;` (for example, `"title: Writing .NET Core console apps&#58 An advanced step-by-step guide"`).
+- **title**: This title will appear in search engine results. The title should end with a pipe (|) followed by the name of the service (for example, see above). The title doesn't need (and probably shouldn't) be identical to the title in your H1 heading. It should be roughly 65 characters (including | SERVICE NAME)
```

Comment:
Also, what example is this referring to?

If it's talking about `title: [ARTICLE TITLE | SERVICE NAME]`, than that doesn't seem very useful (and is not really an example).

If it's talking about `title: Writing .NET Core console apps&#58 An advanced step-by-step guide`, then that doesn't seem like a good example, since it doesn't contain pipe.